nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -422,12 +415,91 @@ buildOperatorCodes(const TfOpIdxMap &map, flatbuffers::FlatBufferBuilder &fbb) {
   return fbb.CreateVector(fb_op_codes);
 }
 
+/**
+ * @brief get Serialized Tensor shape
+ *
+ * @param dim Tensor dimension to convert
+ * @param type tflite Tensor type
+ * @return flatbuffers::Offset<flatbuffers::Vector<int32_t>>
+ */
+flatbuffers::Offset<flatbuffers::Vector<int32_t>>
+buildTesorShape(const TensorDim &dim, flatbuffers::FlatBufferBuilder &fbb,
+                int effective_dimension, bool is_signature = false) {
+  std::vector<int32_t> fb_dimension;
+
+  /// batch dimension is always considered with signature
+  if ((effective_dimension >> 3) & 1) {
+    fb_dimension.push_back(is_signature ? -1 : dim.batch());
+  }
+
+  for (unsigned int i = 1; i < MAXDIM; ++i) {
+    if (((effective_dimension >> (MAXDIM - i - 1)) & 1) == 0) {
+      continue;
+    }
+    fb_dimension.push_back(dim.getTensorDim(i));
```

Comment:
does the transpose needs to happen here for NCHW/NHWC? if yes, add comment for TODO. If handled else where, add comment that handled else where.

Reply:
transpose is handled elsewhere, this is where that weird `varGradTranspose` comes into.
This commit doesn't contain that part yet.

Reply:
alright. Please just add a small comment that transpose is already done or will be handled later at ...

Reply:
Added to the doxygen note.